app/jobs/backfill_visitor_ids_job.rb

```diff
@@ -0,0 +1,11 @@
+class BackfillVisitorIdsJob < ApplicationJob
+  queue_as :default
+
+  def perform
```

Comment:
was thinking this could be done in the console but didn't want to do it on my own! let me know what you think is the best way to do this

Reply:
Definitely big 👍 for not doing things in the console. Going through code review is a good idea, and can prevent typos in production that can affect user data. I think a job makes sense for this kind of thing, also just a simple class like I did here worked pretty well: https://github.com/codeforamerica/vita-min/commit/2fb2916e5fa7c19946182a56e476af7d50fbc45c